src/routes/(admin)/users/[action]/[id]/UserForm.svelte

```
@@ -3,16 +3,18 @@
 	import { Input } from '$lib/components/ui/input/index.js';
 	import { Checkbox } from '$lib/components/ui/checkbox/index.js';
 	import * as Select from '$lib/components/ui/select/index.js';
-	import { userSchema, type FormSchema } from './schema';
+	import { createUserSchema, editUserSchema, type FormSchema } from './schema';
 	import { type SuperValidated, type Infer, superForm } from 'sveltekit-superforms';
```

ESLint

[error] 'SuperValidated' is defined but never used.

(@typescript-eslint/no-unused-vars)